bin/now-deploy.js

```diff
@@ -569,8 +569,14 @@ const assignAlias = async (autoAlias, token, deployment) => {
     }
   }
 
-  // Throw an error if it doesn't
+  // If it doesn't
```

Comment:
Just trying to keep the same comment pattern. From:

```
// Throw an error if it doesn't
```

to

```
// If it doesn't
...
// Check something
...
// Else, throw an error
```

Happy to change it to anything else :)